token-swap/js/client/token-swap.js

```diff
@@ -328,23 +380,16 @@ export class TokenSwap {
   }
 
   /**
-   * Swap the tokens in the pool
+   * Swap token A for token B
    *
-   * @param authority Authority
-   * @param source Source account
-   * @param swapSource Base account to swap into, must be a source token
-   * @param swapDestination Base account to swap from, must be a destination token
+   * @param source Source token account
    * @param destination Destination token account
-   * @param tokenProgramId Token program id
-   * @param amount Amount to transfer from source account
+   * @param amountIn Amount to transfer from source account
+   * @param minimumAmountOut Minimum amount to send
```

Comment:
This param should be "Minimum amount to receive"

Reply:
Isn't `amountIn` the amount the program's receiving, and `minimumAmountOut` the amount the program's sending?

Reply:
Ah, my mistake, I misunderstood.  Yes, `minimumAmountOut` is how much the program sends out, but I thought the comment implied how much the user was sending in.  Since this `token-swap` js package is user-facing, the parameters should be framed from the point of view of the user, and not the program.  The user is sending in `amountIn`, and receiving `minimumAmountOut`.  Does that make sense?

Reply:
yep!